app/controllers/pages_controller.rb

```diff
@@ -29,6 +34,46 @@ def render_footer?
 
   private
 
+  def features_and_profiles
+    feature_records = Feature.published_feature.order(:position) if !current_region
+
+    results_array = []
+    feature_records.each do |feature|
+      results_array << {
+                      title: feature.title,
+                      description: feature.description,
+                      profiles: profiles_array(feature.profiles)
+                    }
+    end
+    results_array
+  end
+
+  def profiles_array(profiles)
```

Comment:
This would be a shorter way to write the method:

```
def profiles_array(profiles)
  profiles.map do |profile|
    {
      id: profile.id,
      fullname: profile.fullname,
      iso_languages: profile.iso_languages,
      city: profile.city,
      willing_to_travel: profile.willing_to_travel,
      nonprofit: profile.nonprofit,
      main_topic_or_first_topic: profile.main_topic_or_first_topic
    }
  end
end
```